components/atoms/EntitlementInfo.js

```diff
@@ -0,0 +1,56 @@
+import PropTypes from "prop-types";
+
+export function EntitlementInfo(props) {
+  return (
+    <div
+      className={`flex flex-col items-center rounded-b-lg shadow-sm border relative w-full align-top ${props.bgColor}`}
+      data-cy={props.dataCy}
+    >
+      <span className=" absolute -top-3  bg-white rounded-full h-5 w-5 xl:h-7 xl:w-7">
+        {props.icon}
+      </span>
+
+      <div
```

Comment:
For accessibility reasons, it would be better to have text in respective tags like a p tag. It also assists screen readers.